### ISSUE TITLE
SystemError: PY_SSIZE_T_CLEAN macro must be defined for '#' formats

### ISSUE THREAD

**Issue by ilyamodder** [dualra1n/dualra1n#74]
I have this error on ios13 branch:

```[*] Saving snapshot
[*] Finished step 1. you can use --dont-create-part in order to dont have to copy and create all again if you needed.
[*] Starting step 2
[*] Fixing firmwares
[*] Patching kernel ...
Reading work/kernelcache...
[NOTE] Image4 payload data is LZSS compressed, decompressing...
SystemError: PY_SSIZE_T_CLEAN macro must be defined for '#' formats
```

It seems that pyimg4 doesn't work with new Python versions (I have 3.11), replacing all `python3 -m pyimg4` with `python3.8 -m pyimg4` fixes the issue.

**Comment by edwin170** [comment]
> I have this error on ios13 branch:
> 
> ```
> [*] Finished step 1. you can use --dont-create-part in order to dont have to copy and create all again if you needed.
> [*] Starting step 2
> [*] Fixing firmwares
> [*] Patching kernel ...
> Reading work/kernelcache...
> [NOTE] Image4 payload data is LZSS compressed, decompressing...
> SystemError: PY_SSIZE_T_CLEAN macro must be defined for '#' formats
> ```
> 
> It seems that pyimg4 doesn't work with new Python versions (I have 3.11), replacing all `python3 -m pyimg4` with `python3.8 -m pyimg4` fixes the issue.


 thank you for say it

**Comment by m1stadev** [comment]
hi, just noticed this. are you sure that you're using the latest PyIMG4 version? this was an issue I fixed in [PyIMG4 0.5](https://github.com/m1stadev/PyIMG4/releases/tag/v0.5).

**Comment by dualra1n** [comment]
> you'll need to include the `-U` flag (e.g. `pip install -U pyimg4`. you may also want to depend on the `compression` extra, since the compression libraries aren't installed by default anymore.

Uh alright thank you so much bro, and the library aren't installed automatically 🥲 but ok I am gonna add it
